app.js

```diff
@@ -26,8 +27,14 @@ const connector = new builder.ChatConnector({
 server.post('/api/messages', connector.listen());
 
 const bot = new builder.UniversalBot(connector, (session) => {
-    //session.send('Welcome to DotABot!');
-    session.send('Sorry, I did not understand \'%s\'. Type \'help\' if you need assistance.', session.message.text);
+
+    if(session.message.text.toLowerCase() === "help") {
```

Comment:
Filed a bug to improve this. Would include that alter, but merging it as is for now.